README.md

```diff
@@ -1,223 +1,203 @@
 install the coloredlogs module using pip. `pip install coloredlogs`
     * There have been issues with coloredlogs installation on Windows.
      We might switch to a different module at some point to enable
      colored logging on Windows as well.
-* [valgrind](http://valgrind.org)
-* [gdb](https://www.gnu.org/software/gdb/)
-
-
-### Installation step-by-step
-
-```
-git clone https://github.com/ARMmbed/icetea.git
-cd icetea
-python setup.py install
-```
-
-**Linux**
-
-In order to run test cases with hardware in Linux without sudo rights:
-
-```
-sudo usermod -a -G dialout username
-Log out & log in back to Linux
-```
 
-This command will add the user 'username' to the 'dialout' group and
-grant the required permissions to the USB ports.
-
-## Usage
+### Usage
 
 To print the help page:
 
 `icetea --help`
 
-All cli parameters are described in [Icetea.md](doc/Icetea.md)
+To list all local testcases from the examples subfolder:
 
-To list all local testcases from the `./testcases` subfolder:
+`icetea --list --tcdir examples`
 
-`icetea --list`
+To print Icetea version:
 
-## Test Case API
+`icetea --version`
 
-To execute a single command line:
+#### Typical use
 
-` execute_command(<dut>, <cmd>, (<arguments>) ) `
+All of the commands described below might also need other options,
+depending on the test case.
 
-or alias `command(...)`
+**Running test cases using the tc argument**
 
+`> icetea --tc <test case name> --tcdir <test case search path>`
 
-```
-dut[number]             #dut index (1..)
-dut[string]             #dut nick name, or '*' to execute command in all duts
-cmd[string]             #command to be executed
-arguments[dictionary]   #optional argument list
-        wait = <boolean>           # whether retcode is expected before continue next command. True (default) or False
-        expected_retcode = <int>    # expected return code (default=0)
-        timeout=<int>              # timeout, if no retcode receive
-```
-
-## Documentation
-HTML documentation for Icetea can be built using sphinx. The source
-for the documentation is located in [doc-source](doc-source).
-For installation of sphinx see [installation](#installation).
+To run all existing test cases from the `examples` folder:
 
-A build script for the documentation is included in build_docs.py.
-It's a simple python scripts that also generates autodoc api documentation.
-Run the script with:
+`> icetea --tc all --tcdir examples`
 
-`python build_docs.py`
+**Running an example test case with hardware**
 
-Currently similar documentation is available in markdown format in
-[doc](doc). This documentation will be phased out and is not maintained
-as well as the sphinx documentation.
+In this example we assume that a compatible board has been connected
+to the computer and an application binary for said board is available.
 
-## Examples
+`> icetea --tc test_cmdline --tcdir examples --type hardware --bin <path to a binary>`
```

Comment:
this assumes that use has icetea clone - where examples lives - need to mention somehow..